tests/security/cc/trustedprograms.pm

```diff
@@ -0,0 +1,62 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary: Run 'trustedprograms' test case of 'audit-test' test suite
+# Maintainer: Liu Xiaojing <xiaojing.liu@suse.com>
+# Tags: poo#95908
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+use audit_test qw(run_testcase compare_run_log);
+use bootloader_setup qw(add_grub_cmdline_settings);
+use power_action_utils "power_action";
+
+sub run {
+    my ($self) = shift;
+
+    select_console 'root-console';
+    
+    # For database testcases
+    zypper_call('in perl-Expect');
+    
+    # Workaround for testcase pam01
```

Comment:
Xiaojing, please add more comments why we need use this workaround.

e.g. : the test cases need restart sshd service many times within a short time.....

Reply:
Fixed